packages/mobile/src/components/user-list/UserCardSkeleton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, Divider, Flex, Paper } from '@audius/harmony-native'
+
+import { Skeleton } from '../skeleton'
+
+export const UserCardSkeleton = () => {
+  return (
+    <Paper border='default'>
+      <Box p='m' pb='s'>
+        <Skeleton
+          style={{ width: '100%', aspectRatio: 1, borderRadius: 1000 }}
+        />
+      </Box>
+      <Flex ph='l' pt='xs' pb='m' gap='xs' alignItems='center'>
+        <Skeleton height={18} width={150} style={{ marginBottom: 6 }} />
```

Comment:
kinda weird to have margin and also flex gap

Reply:
yeah super weird, its trying to mimic line height fwiw, can add a comment